src/Common/precompiled.h

```diff
@@ -393,7 +418,7 @@ class RunAtCemuBoot // -> replaces this with direct function calls. Linkers othe
 template<typename T>
 bool future_is_ready(std::future<T>& f)
 {
-#ifdef __clang__
+#if defined(__clang__) || defined(__GNUC__)
```

Comment:
Clang defines `__GNUC__` already

Reply:
I suppose `__clang__` or `__GNUC__` is somewhat redundant. Changed to just `__GNUC__` in https://github.com/cemu-project/Cemu/commit/71af6e54f44339ab9151c878335a0dfdd1bd1b05